src/SensorStation.ino

```diff
@@ -0,0 +1,293 @@
+/*
+This script represents a sensor station with three sensors attached:
+  * DHT22 for humidity and temperature,
+  * TSL2561 for luminosity,
+  * CCS811 for eCO2 and TVOC
+
+The station reads the sensor data in regular interval and publishes them to MQTT topics.
+
+Run this script on a ESP8266 board.
+*/
+
+#include <Wire.h>
+#include "ccs811.h"
+#include "DHT.h"
+#include <Adafruit_Sensor.h>
+#include <Adafruit_TSL2561_U.h>
+#define ARDUINOJSON_USE_LONG_LONG 1
+#include <ArduinoJson.h>
+#include <NTPClient.h>
+#include <tuple>
+#include <WiFiUdp.h>
+#include <string>
+using namespace std;
+#include <ESP8266WiFi.h>
+#include <PubSubClient.h>
+#include "wifi_secrets.h"
+#include "mqtt_secrets.h"
+#include "sensor_station.h"
+
+const long utcOffsetInSeconds = 3600;
+
+std::string MQTT_CLIENT_PREFIX = "sensor_station_";
+std::string STATION_ID_STR = STATION_ID;
+std::string MQTT_CLIENT_ID_STR = MQTT_CLIENT_PREFIX + STATION_ID_STR;
```

Comment:
you are already using std namespace, so you don't need to have std:: in front of string

... although you might want to [look more into using namespace std](https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice)